VirtoCommerce.LiquidThemeEngine/Converters/JsonPageConverter.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json.Linq;
+using VirtoCommerce.LiquidThemeEngine.Objects;
+using StorefrontModel = VirtoCommerce.Storefront.Model.StaticContent;
+
+namespace VirtoCommerce.LiquidThemeEngine.Converters
+{
+    public static class JsonPageConverter
```

Comment:
Due that fact what we can avoid of usage the special model types for  designer pages (read below) this converters can be removed and replaced to simple Liquid filter for parsing JSON of page body. See the example https://github.com/VirtoCommerce/vc-storefront-core/blob/dev/VirtoCommerce.LiquidThemeEngine/Filters/CommonFilters.cs#L21 

And then resulting liquid template for designer page will looks some like that

`templates/designer-page.liquid`
```HTML

{% assign json_page =  page.content | read_designer_page %}

{% for item in json_page.blocks -%}
    {% assign templateName = "json-"|append:item.type -%}
    {% include templateName, block: item -%}
{% endfor -%}
```

Reply:
add `ParseJson` filter to `CommonFIlters` class.